File: .ipynb_checkpoints/aa-checkpoint.py
f=open('aaindex1.dat', 'r')
g=open('index.csv','w')
fragI=0
raw=''
ID=''
while True:
	data = f.readline()
	data = data.rstrip()
	if data.startswith('I'):
		fragI = 1
	elif data.startswith('H'):
		ID=data
	elif fragI==1 and not data.startswith('//'):
		raw+=data
	elif data.startswith('//'):
		para = raw.split()
		raw=''
		fragI=0
	elif data == '':
		break

File: .ipynb_checkpoints/aa-checkpoint.pl

open(AA, "<aaindex1.dat");

$count=0;
$I=0;

while(<AA>){
	if($_ =~ /^I/){
		$I=1;
	}
	if($_ =~ /^\s\s/ && $I==1){
		$value.=substr($_,0,100);
	}
	if($_ =~ /^\/\//){
		@index=split(/\s/,$value);
		for($j=0;$j<@index;$j++){
			printf $index[$j];
		}
		printf "\n";
		$count++;
		$value="";
		$I=0;
	}
}
printf $count."\n";
print chr(7);	#終了時に音が鳴ります
